### DISCUSSION TITLE
Options configuration for mobile

### DISCUSSION THREAD

**Discussion by suriya-one** [add2cal/add-to-calendar-button#641] [Q&A] [answered]
I'm currently using the button like this:

```html
<add-to-calendar-button
  name="calendar name"
  subscribe
  icsFile="ics file url"
  options="'Apple','Google','iCal','Outlook.com','Microsoft365'"
  optionsMobile="'Google','iCal','Outlook.com','Microsoft365'"
  buttonsList
  buttonStyle="round"
></add-to-calendar-button>
```

Even when using `optionsMobile` and `optionsIOS` I only get apple and google buttons on IOS and google and ical buttons on Android. Are the outlook and microsoft365 not available for mobile?

**Comment by jekuer** [comment, marked as answer]
Not in the "subscribe" case. Unfortunately, Microsoft does not play nice there.
Might add this at the docs.
